exporter/otlpexporter/config_test.go

```diff
@@ -134,6 +134,17 @@ func TestUnmarshalInvalidConfig(t *testing.T) {
 func TestValidDNSEndpoint(t *testing.T) {
 	factory := NewFactory()
 	cfg := factory.CreateDefaultConfig().(*Config)
-	cfg.Endpoint = "dns:///backend.example.com:4317"
+	cfg.Endpoint = "dns://backend.example.com:4317"
 	assert.NoError(t, cfg.Validate())
 }
+
+func TestSanitizeEndpoint(t *testing.T) {
```

Comment:
Can you add a test case for `/` somewhere else in the host?

Reply:
done